astropy/units/quantity.py

```diff
@@ -107,7 +117,6 @@ def __next__(self):
 
     next = __next__
 
-
```

Comment:
There should be 2 lines between classes and functions. Please fix PEP8 warnings reported at https://travis-ci.org/astropy/astropy/jobs/348076730 (I just restarted the job because Travis has problem displaying the log).